### DISCUSSION TITLE
It makes my day brighter

### DISCUSSION THREAD

**Discussion by qortex** [cyrilletuzi/vscode-angular-schematics#624] [General]
Thanks for this pleasing extension.

Adding a component is otherwise always an irritating experience (have to get the dreaded `--path` option right, a challenge in a large project).

I don't have any pain adding a new component, and thus do not hesitate to add small helper components, making my code healthier :)

Thanks!

**Comment by cyrilletuzi** [comment]
Hi Michael @qortex,

Thanks a lot for taking time to show your love, it means a lot to me. 👍🏻

I'm in Paris too, so if your company needs help on Angular (training, consulting...), just let me know. :)
